data/src/main/kotlin/com/lightteam/modpeide/data/repository/LocalRepository.kt

```diff
@@ -39,8 +39,12 @@ class LocalRepository(
     private val filesystem: Filesystem
 ) : DocumentRepository {
 
+    private val encodingForOpeningAutoDetect: Boolean
+        get() = preferenceHandler.getEncodingForOpening().get().trim().isEmpty()
```

Comment:
Please do all transformations like `trim()` in `PreferenceHandler` if it's possible. But I think in this case it's unnecessary